sale_order_general_discount/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+The discount is not aligned with the totalization fields because they are added with a widget.
+For this to be aligned, it would be necessary to investigate how the widget works, which would
+complicate the migration of the module more than it should.
```

Comment:
With my suggestion I'd say this warning can go away

Reply:
I am agree with you.